### ISSUE TITLE
Feature Request: Note Variable

### ISSUE THREAD

**Issue by uroybd** [joplin/plugin-templates#42]
I think an easy way to select a note as a variable can be very useful.

For example, I export all my notes after I read a book from my reading app and put them in Joplin.
I often write reviews and I now have a template for that. If I could select a note as a template field it could've been linked automatically with my review for reference.

**Comment by nishantwrp** [comment]
Yeah, that could be done. But I guess that would be useless if the dropdown doesn't support autocomplete. I am actually looking for a nice and easy way to implement a dropdown with autocomplete. This can be discussed after that.
